ingestion/src/metadata/ingestion/source/database/dbt/metadata.py

```diff
@@ -890,6 +890,40 @@ def process_dbt_descriptions(self, data_model_link: DataModelLink):
                     f"to update dbt description: {exc}"
                 )
 
+    def process_dbt_owners(self, data_model_link: DataModelLink):
+        """
+        Method to process DBT owners using patch APIs
+        """
+        table_entity: Table = data_model_link.table_entity
+        logger.debug(
+            f"Processing DBT owners for: {table_entity.fullyQualifiedName.root}"
+        )
+        if table_entity:
```

Comment:
this `if` might need to happen before the `logger.debug` no? Otherwise the table.fqn would fail

Reply:
okay